ionoscloud/resource_nic.go

```diff
@@ -280,6 +317,12 @@ func NicSetData(d *schema.ResourceData, nic *ionoscloud.Nic) error {
 				return fmt.Errorf("error setting dhcp %w", err)
 			}
 		}
+		log.Printf("DHCPV6: %v", nic.Properties.Dhcpv6)
```

Comment:
any log printf should have a [INFO] or [DEBUG] at the beginning. Hashicorp hijacks the prints and displays them correctly

Reply:
removed unnecessary logging